hasher-matcher-actioner/terraform/fetcher/variables.tf

```diff
@@ -46,3 +46,9 @@ variable "log_retention_in_days" {
   description = "How long to retain cloudwatch logs for lambda functions in days"
   type        = number
 }
+
+variable "fetch_frequency_min" {
+  description = "How many minutes to wait between queries to ThreatExchange for updates"
+  type        = number
+  default     = 15
```

Comment:
Thinking about this number makes me wonder about how a long-running fetcher run interacts with a subsequent run. Is there a CloudWatch feature that understands the concept of not wanting to run two instances of the fetcher lambda concurrently? At some point we need to know what the current behavior is and if necessary we need to prevent unexpected behavior caused by concurrency issues.

Reply:
good call out